aws-athena/views/location-vw_pin10_location.sql

```diff
@@ -111,34 +111,34 @@ SELECT
     other.misc_subdivision_id,
     other.misc_subdivision_data_year
 
-FROM spatial.parcel AS pin
-LEFT JOIN location.census
+FROM {{ ref('parcel') }} AS pin
+LEFT JOIN {{ ref('location_census') }} AS census
```

Comment:
Here's an example of a table (`location.census`) where we're using an alias (`location_census`) to act as a unique identifier for the `ref()` function without having to change the name of the table itself. We'll see below that this corresponds to a change in the model's filename, in addition to an `alias` attribute being added to the query via a `config()` call.